src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -3,8 +3,9 @@
 		{{ loanUseFiltered }}
 		<kv-text-link
 			v-if="loanId && showLearnMore"
-			:to="`/lend/${loanId}`"
+			:to="showLoanDetails ? '' : `/lend/${loanId}`"
```

Comment:
I believe you can pass undefined to exclude the prop altogether, so this line could be: `:to="showLoanDetails ? undefined : `/lend/${loanId}`"` and then if showLoanDetails this prop will not be passed at all

Reply:
The click on this text link ultimately needs to emit an event we can capture... @michelleinez is it already bubbling up something you can use to show the In Context Borrower Profile? Or, should it actually call `showLoanDetails` ?

Reply:
That is good advice though @eddieferrer for the right situation

Reply:
I'm changing this to emit an event now.  The to field will have a boolean instead which defaults to false so that the default behavior doesn't change when the prop is undef.